src/components/home/home.js

```diff
@@ -0,0 +1,62 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { getCategories } from '../../redux/categories/categories';
+import LoadingPage from '../loadingPage/loadingPage';
+import home from '../../assets/home.jpg';
+import './home.css';
+
+const HomeList = () => {
+  const dispatch = useDispatch();
+  const categories = useSelector((state) => state.categories);
+  useEffect(() => {
+    dispatch(getCategories);
+  }, []);
+
+  return (
+    <>
+      {categories.length === 0
+        ? (
+          <div className="loading">
+            <LoadingPage />
+          </div>
+        )
+        : (
+          <div className="articles_container">
+            <img className="home-img" src={home} alt="home" />
+            <div className="presentation">
+              <h2 className="open">We are open!</h2>
+              We are PREMIUM STORE, we deliver the most
+              trending articles just for exclusive customers like you!
+            </div>
+            <div className="categories-selector">
+              <h4>Categories:</h4>
+            </div>
```

Comment:
I do not agree with this, because creating the third page was not in the requirements, and the main goal of the app is to use Dynamic Url to filter categories, and my second page is showing details of my categories which are products. @Davidosky007